### PR TITLE
Deliverability Center EA - MERGE 4/20

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #** BD-1919**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+nav_title: Deliverability Center
+article_title: Deliverability Center
+page_order: 4
+description: "This reference article covers how to set up the Deliverability Center, a feature that allows marketers to view their email domains and IP reputations, and understand their email deliverability."
```
Comment (jaydubbyasee) [inline _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md:5]: Small change here. I think it's good to clarify that it's specific to sending domain reputation and not things like tracking domains.

```suggestion
description: "This reference article covers how to set up the Deliverability Center, a feature that allows marketers to view their email sending domains and IP reputations, and understand their email deliverability."
```

File: _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Integrate with Google Postmaster
+
+Before setting up your Deliverability Center, check that your domains have been [added to the Gmail Postmaster Tools][4].
+
+Follow these steps to integrate with Google Postmaster and set up your Deliverability Center:
+
+1. Go to the **Overview** page in the Braze dashboard.
+2. Click **Email Performance** and select the **Deliverability Center** tab. <br>![][3]
+3. Click **Connect with Google Postmaster**. Select your Google Account, and click the **Allow** button to allow Braze to view email traffic metrics for the domains registered with the Postmaster Tools.
+4. Your verified domains will display in the Deliverability Center. <br>![][5]
```
Comment (jaydubbyasee) [inline _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md:26]: After successfully installing we schedule a sync job to pull reputation and error data for the last 30 days. The data may not be immediately available and could take several minutes to propagate.

File: _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Email deliverability is at the core when it comes to measuring the success of a messaging campaign. Using the Deliverability Center in the Braze dashboard, you can view your domains by **IP Reputation** or **Delivery Errors** to discover and troubleshoot any potential issues with email deliverability. 
+
+## Integrate with Google Postmaster
+
+Before setting up your Deliverability Center, check that your domains have been [added to the Gmail Postmaster Tools][4].
+
+Follow these steps to integrate with Google Postmaster and set up your Deliverability Center:
+
+1. Go to the **Overview** page in the Braze dashboard.
```
Comment (jaydubbyasee) [inline _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md:23]: This integration was actually built as an Alloys integration so they can also access it from Partner Integrations page and search the Google Postmaster tile.

image


We might also want to add documentation in the Technology Partners page that backlinks to this page.
image

**Review by jaydubbyasee — COMMENTED** [review]
Hey Lydia this looks really good! Mostly some minor clarifications needed and some revision needed related to initial installation instructions. Since this actually a Partner Integration, it may make sense to highlight that they should navigate to Partner Integrations to enable the feature.

File: _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+2. Click **Email Performance** and select the **Deliverability Center** tab. <br>![][3]
+3. Click **Connect with Google Postmaster**. Select your Google Account, and click the **Allow** button to allow Braze to view email traffic metrics for the domains registered with the Postmaster Tools.
+4. Your verified domains will display in the Deliverability Center. <br>![][5]
+
+You can also access Google Postmaster in the Braze dashboard by navigating to **Technology Partners > Google Postmaster**. After intergating, Braze pulls reputation and error data for the last 30 days. The data may not be immediately available and could take several minutes to populate.
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md:32]:
```suggestion
You can also access Google Postmaster in the Braze dashboard by navigating to **Technology Partners > Google Postmaster**. After integrating, Braze pulls reputation and error data for the last 30 days. The data may not be immediately available and could take several minutes to populate.
```

**Review by KellieHawks — APPROVED** [review]
Looks great!

**Pull request merged**